helix-term/src/ui/statusline.rs

```diff
@@ -280,6 +281,19 @@ where
     }
 }
 
+fn render_file_line_ending<F>(context: &mut RenderContext, write: F)
+where
+    F: Fn(&mut RenderContext, String, Option<Style>) + Copy,
+{
+    use helix_core::LineEnding::*;
+    let line_ending = match context.doc.line_ending {
+        Crlf => "CRLF",
+        LF => "LF",
+    };
```

Comment:
There can be additional line endings if the `unicode-lines` feature flag is enabled (even though [not all of them](https://github.com/helix-editor/helix/blob/791bf7e50a19bcf7612788deb7514847089cb976/helix-core/src/line_ending.rs#L135) can be used as actual document line endings, but anyways):

https://github.com/helix-editor/helix/blob/791bf7e50a19bcf7612788deb7514847089cb976/helix-core/src/line_ending.rs#L10-L25